src/components/FilterButton/FilterButton.style.ts

```diff
@@ -0,0 +1,23 @@
+import styled from '@emotion/styled';
+import theme from '@styles/theme';
+
+interface Props {
+  color: string;
+  size: string;
+  fontSize: string;
+  fontColor: string;
+  margin: string;
+}
+
+export const Container = styled.button<Props>`
+  position: relative;
+  display: inline-block;
+  border: none;
+  border-radius: 50%;
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  font-size: ${({ fontSize }) => fontSize};
+  margin-right: ${({ margin }) => (margin === 'default' ? '0' : margin)};
+  background-color: ${({ color }) => (color === 'primary' ? `${theme.color.green200}` : color)};
+  color: ${({ fontColor }) => (fontColor === 'primary' ? `${theme.color.background}` : fontColor)};
```

Comment:
```suggestion
  background-color: ${({ color, theme }) => (color === 'primary' ? `${theme.color.green200}` : color)};
  color: ${({ fontColor, theme }) => (fontColor === 'primary' ? `${theme.color.background}` : fontColor)};
```
여기도 이렇게 수정하면 좋을 것 같습니다!

Reply:
반영해쑴다~~!